### PR TITLE
feat: add t function, issue 10474

### LINKED ISSUE #10474: t

### PR DESCRIPTION
**Pull request by ChunchangShao**
Close #10474

### CODE REVIEW

**Review by fspyridakos — CHANGES_REQUESTED** [review]
Thanks for the contribution! I left a comment for one more addition to the function. Can you also add the test for it? Thanks

**Comment by fspyridakos** [comment]
Also, I will link this to the corresponding issue, but in future you should do this yourself by adding a comment ‘Close #Issue_number’, as explained [here](https://lets-unify.ai/ivy/contributing/1_the_basics.html#todo-list-issues)

File: ivy/functional/frontends/torch/indexing_slicing_joining_mutating_ops.py [hunk 1]
```diff
@@ -125,6 +125,13 @@ def swapdims(input, dim0, dim1):
 def transpose(input, dim0, dim1):
     return ivy.swapaxes(input, dim0, dim1)
 
+@to_ivy_arrays_and_back
+def t(input):
+    if input.ndim > 2:
+        raise ivy.exceptions.IvyException("t(input) expects a tensor with <= 2 dimensions, but self is %dD" % input.ndim)
+
```
[the remark this answers is not in the document]
Comment (ChunchangShao) [reply ivy/functional/frontends/torch/indexing_slicing_joining_mutating_ops.py:132]: Hi, thank you for your help. I add the if and test function as your request.

**Comment by fspyridakos** [comment]
Looks good, thanks for your contribution! Just made a small formatting change myself

**Pull request merged**